catalog/rest/catalog-rest-endpoint/src/main/java/org/codice/ddf/endpoints/rest/RESTEndpoint.java

```diff
@@ -519,7 +519,7 @@ public Response getDocument(@PathParam("sourceid") String sourceid, @PathParam("
                 LOGGER.warn(exceptionMessage, e);
                 throw new ServerErrorException(exceptionMessage, Status.INTERNAL_SERVER_ERROR);
             } catch (CatalogTransformerException e) {
-                String exceptionMessage = "Unable to transform Metacard.  Try different transformer: ";
+                String exceptionMessage = "Unable to transform Metacard : \n" + e.getMessage();
```

Comment:
I have always thought there was a middle ground here.  We don't want to send back a stack trace to the client.  OTOH, it would be nice to provide something more meaningful that "Unable to transformer metacard."